readme.md

```diff
@@ -0,0 +1,63 @@
+<img width="659" alt="image" src="https://user-images.githubusercontent.com/13199349/117996907-d6ecf900-b342-11eb-90e6-20cd78557a99.png">
+
+# Glossary
+An application for patients with cancer that are in remission, where they can find guidance and support from similar people.
+The app is intended to help people get back to their daily lives.
+
+**Core functionalities**
+* As a user, I want information about relevant themes, so that I would know what to expect after the treatment process.
+* As a user, I want to have 1 on 1 contact with similar people, so that I can reflect on my experiences, and further develop my coping strategies.
+
+# Technicalities
+
+## Installation
+```BASH
+# Navigate to folder
+git clone https://github.com/sjagoori/bloom.git
+cd bloom
+# CLI into editor of choice, open two terminal and run the following
+cd frontend && npm run dev
+cd backend && npm start server.js
+# Happy coding! 🎉
+```
+
+## Code Conventions
+
+Function over variables. We don't want to save functions in variables as that can.
+
+<span style="color: red">To be continued.</span>
+
+## Git Conventions
+
+|Symbol|What does it do|
+|:-:|:--|
+| `+` |Commit when successful.|
+| `-` |Commit a deletion, be it a file or code.|
+| `~` |Commit when something is broken or changed, such as logic or formatting.|
+|`[WIP]`|Commit when something is unfinished. <span style="color: red">Refrain from committing unfinished work.</span> |
+
+## Tooling and more
+
+**[Next.js](https://nextjs.org/), The React Framework for Production**  
+Next.js gives you the best developer experience with all the features you need for production: hybrid static & server rendering, TypeScript support, smart bundling, route pre-fetching, and more. No config needed.
+
+**[MongoDB](https://www.mongodb.com/)**  
+The database for modern napplications. MongoDB is a general purpose, **document-based**, distributed database built for modern application developers and for the cloud era. 
+
+**[Socket.io](https://socket.io/)**  
+Socket.IO enables real-time, bidirectional and event-based communication.
+It works on every platform, browser or device, focusing equally on reliability and speed. 
+
+### Other dependencies
+|Name|Purpose|
+|:--|:--|
+|[dotenv]()|Storing sensitive credentials in a safe environment.|
+|[Socket.io](https://www.npmjs.com/package/socket.io|Two way real time communication|
```

Comment:
fix the links here

Reply:
Fixed the links in commit 8fabfd6.

* Add link to dotenv.
* Fix markdown error in the second link.